WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModelTests.swift

```diff
@@ -569,5 +701,8 @@ private extension DefaultProductFormTableViewModelTests {
                                                        comment: "Format of the Maximum Quantity setting (with a numeric quantity) on the Quantity Rules row")
         static let groupOfFormat = NSLocalizedString("Group of: %@",
                                                        comment: "Format of the Group Of setting (with a numeric quantity) on the Quantity Rules row")
+        static let noTrialPeriod = NSLocalizedString("defaultProductFormTableViewModelTests.noFreeTrial",
+                                                   value: "No trial period",
+                                                   comment: "Display label when a subscription has no trial period.")
```

Comment:
Is it possible to use the same translated string from the code? Since we are marking this string with a different key, there's a small chance that the translation differs from the one in the code.

Reply:
Nice catch. I addressed this by making free trial strings public. https://github.com/woocommerce/woocommerce-ios/pull/11173/commits/9f156eb19c72b3bf15ab05f8c5b8ac3af135e95d